pmg/templates/index.html

```diff
@@ -46,7 +47,7 @@
           <div class="row">
             {% for meeting in featured_content.committee_meetings %}
               <div class="col-xs-12 col-sm-6 col-md-3">
-                <a href="{{ url_for('committee_meeting', event_id=meeting.id) }}">
+                <a href="{{ url_for('committee_meeting', event_id=meeting.id) }}?utm_source=homepage&utm_medium=feature">
```

Comment:
Nice. I think you can include extra params as part of ``url_for``, like this:

``url_for('committee_meeting', event_id=meeting.id, utm_source='homepage', utm_medium='featured')``